common/changes/office-ui-fabric-react/bugfix-colorpicker-hue-change_2018-08-09-13-28.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Fixed an issue #5871",
```

Comment:
Please describe the issue here rather than referencing an issue number. These comments go into release notes so this should say what the issue is and what you changed. Something like, e.g. "ColorPicker: allow state update when hue value changes".